06_distributed_advanced.ipynb

```diff
@@ -220,6 +233,7 @@
    },
    "outputs": [],
    "source": [
+    "from dask import distributed\n",
```

Comment:
We've been avoiding using the distributed namespace directly in examples (it is an overly-generic name)

Instead I recommend `from dask.distributed import progress`

Reply:
Done.